### PR TITLE
Sor10874/readmes/41 45

### PR DESCRIPTION
**Pull request by sorenoid**
this PR covers 

MapReferenceScale 
MapRotation 
MapSpatialReference (android set-map-spatial-reference)
MinMaxScale (android set-min-max-scale)
MobileMapSearchAndRoute

### CODE REVIEW

File: java/set-min-max-scale/README.md [hunk 1]
```diff
@@ -1,22 +1,30 @@
 # Min max scale
```
Comment (hud10837) [inline java/set-min-max-scale/README.md]: This is called "Set min max scale"
```suggestion
# Set min max scale
```

File: java/map-reference-scale/README.md [hunk 2]
```diff
@@ -1,28 +1,35 @@
...
 Setting a reference scale on an `ArcGISMap` fixes the size of symbols and text to the desired height and width at that scale. As you zoom in and out, symbols and text will increase or decrease in size accordingly. When no reference scale is set, symbol and text sizes remain the same size relative to the `MapView`.
 
 Map annotations are typically only relevant at certain scales. For instance, annotations to a map showing a construction site are only relevant at that construction site's scale. So, when the map is zoomed out that information shouldn't scale with the `MapView`, but should instead remain scaled with the `ArcGISMap`. 
 
 ## How to use the sample
-* Use the spinner at the top to set the map's reference scale.
-* Use the menu checkboxes to set which feature layers should scale according to the reference scale.
-* Click the button at the bottom to set the map scale (zoom) to the reference scale.
+
+* Use the drop box at the top to set the map's reference scale (1:500,000 1:250,000 1:100,000 1:50,000).
```
[the remark this answers is not in the document]
Comment (sorenoid) [reply java/map-reference-scale/README.md]: i was wondering about that but was a bit timid.

**Comment by sorenoid** [comment]
ok pinging @Rachael-E for 2nd review!

File: java/mobile-map-search-and-route/README.md [hunk 3]
```diff
@@ -1,21 +1,36 @@
...
+
+A field worker might download a mobile map package to support their operations while working offline.
 
 ## How to use the sample
-The sample loads maps from a `MobileMapPackage` and displays the first map in the map package (index 0) on the starting activity (MobileMapViewActivity). All maps contained within the map package are shown as a recycler view list in a seperate activity (MapChooserActivity). You can tap on a single map to load its contents to the main `MapView` on MobileMapViewActivity. The map chooser menu indicates if the map has `transportationNetworks` datasets (for routing) or a `LocatorTask` (for geocoding). If the mobile map supports routing or geocoding, you can tap on the map for results.
+
+A list of maps from a mobile map package will be displayed. If the map contains transportation networks, the list item will have a navigation icon. Click on a map in the list to open it. If a locator task is available, click on the map to reverse geocode the location's address. If transportation networks are available, a route will be calculated between geocode locations.
```
Comment (Rachael-E) [inline java/mobile-map-search-and-route/README.md]: that ol' chestnut "click" has snuck in here twice! Click -> Tap :)

File: java/mobile-map-search-and-route/README.md [hunk 4]
```diff
@@ -1,21 +1,36 @@
...
 
 ## How it works
-The sample loads a map from a `MobileMapPackage` to instantiate a map package object. It uses the `LocatorTask` property on `MobileMapPackage` to check if the package supports geocoding. It uses the `transportationNetworks` property on each mobile map to see if routing is supported. The logic for routing and geocoding is similar to the one used in the individual routing and geocoding samples.
+
+1.  Create a `MobileMapPackage` using `MobileMapPackage(path).loadAsync()`.
+2.  Get a list of maps inside the package using the `MobileMapPackage.getMaps()`.
```
Comment (Rachael-E) [inline java/mobile-map-search-and-route/README.md]: just so we're consistent across our readmes, the second time an object is referenced after being "instantiated" in a previous line, we can then just refer to it in camel case as if it was a variable 
```suggestion
2.  Get a list of maps inside the package using the `mobileMapPackage.getMaps()`.
```

File: java/mobile-map-search-and-route/README.md [hunk 5]
```diff
@@ -1,21 +1,36 @@
...
+3.  If the package has a locator, access it using `MobileMapPackage.getLocatorTask()`.
```
Comment (Rachael-E) [inline java/mobile-map-search-and-route/README.md]:
```suggestion
3.  If the package has a locator, access it using `mobileMapPackage.getLocatorTask()`.
```

File: java/mobile-map-search-and-route/README.md [hunk 6]
```diff
@@ -1,21 +1,36 @@
...
-* LocatorTask
-* GeoCode
+* ReverseGeocodeParameters
 * Route
-* NetworkAnalysis
-* Callout
+* RouteParameters
+* RouteResult
+* RouteTask
+* TransportationNetworkDataset
```
Comment (Rachael-E) [inline java/mobile-map-search-and-route/README.md]: looks like we don't have this class in our API

File: java/set-map-spatial-reference/README.md [hunk 7]
```diff
@@ -1,22 +1,38 @@
...
+Pan and zoom around the map. Observe how the map is displayed using the World Bonne spatial reference.
 
 ## How it works
-1. Create a `MapView`.
-1. Create an `ArcGISMap` passing a specific `SpatialReference` as an argument.
 
-NOTE: Not all layer types can be reprojected, like `ArcGISTiledLayer`, which will fail to draw if their spatial reference is not the same as the `ArcGISMap`'s spatial reference.
+1. Instantiate an `ArcGISMap` object using a spatial reference e.g. `ArcGISMap(SpatialReference.create(54024))`.
+2. Instantiate a `Basemap` object using an `ArcGISMapImageLayer` object.
+3. Set the base map to the map.
```
Comment (Rachael-E) [inline java/set-map-spatial-reference/README.md]: we should have a code instruction in this workflow in this step too

**Comment by sorenoid** [comment]
over to @hud10837

**Comment by sorenoid** [comment]
@Rachael-E @hud10837 I believe it's all sorted on my end.

**Comment by sorenoid** [comment]
@hud10837 is it ok to merge this one?

**Pull request merged**